internal/schema/schemas/draft/posit-publishing-schema-v3.json

```diff
@@ -116,6 +116,12 @@
           "default": "pip",
           "description": "Package manager that will install the dependencies. If package-manager is none, dependencies are assumed to be pre-installed on the server. The default is 'pip'.",
           "examples": ["pip", "conda", "pipenv", "poetry", "none"]
+        },
+        "python_requires": {
```

Comment:
Since this attribute is within the [python] section, can we simply call it `requires` rather than `python_requires`?

Reply:
The name `python_requires` is to be consistent with the option itself has, but maybe using `requires_python` which is how instead the option is named in the `.toml` could be an improvement?

I'd like to keep the option bound to the names it has in the actual python config files

Reply:
Thanks for making this change.